EFCore-Adapter/CasbinDbAdapter.cs

```diff
@@ -99,56 +99,53 @@ public void RemoveFilteredPolicy(string sec, string ptype, int fieldIndex, param
 
         public void SavePolicy(Model model)
         {
-            List<CasbinRule<TKey>> lines = new List<CasbinRule<TKey>>();
-            if (model.Model.ContainsKey("p"))
-            {
-                foreach (var kv in model.Model["p"])
-                {
-                    var ptype = kv.Key;
-                    var ast = kv.Value;
-                    foreach (var rule in ast.Policy)
-                    {
-                        var line = savePolicyLine(ptype, rule);
-                        lines.Add(line);
-                    }
-                }
-            }
-            if (model.Model.ContainsKey("g"))
-            {
-                foreach (var kv in model.Model["g"])
-                {
-                    var ptype = kv.Key;
-                    var ast = kv.Value;
-                    foreach (var rule in ast.Policy)
-                    {
-                        var line = savePolicyLine(ptype, rule);
-                        lines.Add(line);
-                    }
-                }
-            }
+            var lines = SavePolicyLines(model);
             if (lines.Any())
             {
                 _context.CasbinRule.AddRange(lines);
                 _context.SaveChanges();
             }
         }
 
-        public void AddPolicy(string sec, string pType, IList<string> rule)
+        public void AddPolicy(string sec, string ptype, IList<string> rule)
         {
-            var line = savePolicyLine(pType, rule);
+            var line = SavePolicyLine(ptype, rule);
             _context.CasbinRule.Add(line);
             _context.SaveChanges();
         }
 
+        public async Task LoadPolicyAsync(Model model)
```

Comment:
Added some new APIs, would be great to add some tests.

Reply:
I thought about it, and I found it can not call those api by `Enforcer` at `Casbin.NET` current version like exist test. If who want to test those apis, it looks need to write a new test which only used the `CasbinDbAdapter`.

Reply:
Ok, I see, we don't need `LoadPolicyAsync` but it's declared in `Casbin.NET`, and can't access from `Casbin.NET`. The code looks correct, I think we can merge first